zero/moment_em_coupling.c

```diff
@@ -148,11 +159,30 @@ gkyl_moment_em_coupling_explicit_advance(const gkyl_moment_em_coupling* mom_em,
       app_accel_s[i] = gkyl_array_cfetch(app_accel[i], cell_idx);
     }
 
-    double *em_arr = gkyl_array_fetch(em, cell_idx);
-    const double *app_current_arr = gkyl_array_cfetch(app_current, cell_idx);
-    const double *app_current1_arr = gkyl_array_cfetch(app_current1, cell_idx);
-    const double *app_current2_arr = gkyl_array_cfetch(app_current2, cell_idx);
-    const double *ext_em_arr = gkyl_array_cfetch(ext_em, cell_idx);
+    double *em_arr = 0;
```

Comment:
I think we should make these lines ternary operators like we have elsewhere in the code:

double *em_arr = em ? gkyl_array_fetch(em, cell_idx) : 0;

I'm not normally one for compactifying lines, but we do this everywhere else in the code and it really is quite nice.